src/attestedclaim/AttestedClaim.ts

```diff
@@ -30,13 +29,15 @@ export default class AttestedClaim implements IAttestedClaim {
    * AttestedClaim.fromObject(JSON.parse(serialized));
    * ```
    */
-  public static fromObject(obj: IAttestedClaim): AttestedClaim {
-    const newAttestedClaim: AttestedClaim = Object.create(
-      AttestedClaim.prototype
-    )
-    newAttestedClaim.request = RequestForAttestation.fromObject(obj.request)
-    newAttestedClaim.attestation = Attestation.fromObject(obj.attestation)
-    return newAttestedClaim
+  public static fromAttestedClaimInterface(obj: IAttestedClaim): AttestedClaim {
```

Comment:
Great name. Since it's static, we could even rename it to `fromInterface` for consistency across classes - @tjwelde any thoughts?

edit: I see now that it's named this way to distinguish it from `fromRequestInterface`. We could still make the above suggested change for brevity.